packages/datetime/src/dateRangePicker.tsx

```diff
@@ -618,7 +618,8 @@ export class DateRangePicker
 
 function getStateChange(value: DateRange,
                         nextValue: DateRange,
-                        state: IDateRangePickerState): IDateRangePickerState {
+                        state: IDateRangePickerState,
+                        props: IDateRangePickerProps): IDateRangePickerState {
```

Comment:
sweet! one final small nit: can you just pass in `contiguousCalendarMonths` directly rather than the whole prop?